src/record/record.js

```diff
@@ -249,17 +250,17 @@ Record.prototype.whenReady = function( callback ) {
  */
 Record.prototype._$onMessage = function( message ) {
 	if( message.action === C.ACTIONS.READ ) {
-		this._clearTimeouts();
+		clearTimeout( this._readTimeout );
```

Comment:
readAck is cleared as a normal ack from server as part of the handle ack workflow

R|A|S|recordName+